src/pages/Calculator/Steps/SpeciesSelection/PlantList.js

```diff
@@ -160,28 +175,100 @@ const PlantList = ({
                       : seed.thumbnail)
                   }
                   alt={seed.label}
-                  sx={{ border: '2px solid green', borderRadius: '1rem' }}
+                  sx={{
+                    border: '2px solid green',
+                    borderRadius: '1rem',
+
+                    ...(seedsSelected.filter((s) => s.label === seed.label).length > 0 && {
+                      border: '6px solid #5992E6',
+                    }),
+                  }}
+
                 />
+                <Typography
+                  sx={{
+                    color: 'green',
```

Comment:
update color

Reply:
To update the color, should I use one of the colors in the stepcs.scss file or a color close to what I used, just in hex?